test/SuperVault.t.sol

```diff
@@ -502,6 +502,38 @@ contract SuperVaultTest is ProtocolActions {
         assertEq(SuperVault(payable(superVaultAddress)).refundReceiver(), deployer);
     }
 
+    //////////////////////////////////////////////////////////////
+    //                     FUZZ TESTS                           //
+    //////////////////////////////////////////////////////////////
+
+    function testFuzz_superVault_rebalance(uint256 finalWeightsOne) public {
+        uint256 finalWeightsOne = bound(finalWeightsOne, 1000, 9000);
+        uint256 finalWeightsTwo = 10_000 - finalWeightsOne;
+
+        vm.startPrank(deployer);
+        SOURCE_CHAIN = ETH;
+
+        uint256 amount = 10_000e6;
+        // Perform a direct deposit to the SuperVault
+        _directDeposit(SUPER_VAULT_ID1, amount);
```

Comment:
Can we increase the complexity of this test by performing a loop with fuzzed deposit amount and rebalance within fuzzed indexes (bounded) also looped?